ddtrace/internal/flare/flare.py

```diff
@@ -270,6 +284,42 @@ def _send_flare_request(self, flare_send_req: FlareSendRequest):
                 if client is not None:
                     client.close()
 
+    def _setup_flare_probes(self, probes: List[FlareProbe]):
+        # Defer importing to avoid eager importing of debugging module
+        from ddtrace.debugging._function.discovery import FunctionDiscovery
```

Comment:
One potential issue we might have here is that we might not find things that are decorated with custom decorators. For example, we might (I haven't tested) fail to find things decorated with `@ModuleWatchdog.*`